source/ed/types.h

```diff
@@ -195,23 +195,25 @@ struct PhysicalMode : public Header, Nameable{
 
 struct Line : public Header, Nameable {
     const static nt::Type_e type = nt::Type_e::Line;
-    std::string external_code;
-    std::string code;
-    std::string forward_name;
-    std::string backward_name;
+    std::string external_code = "";
+    std::string code = "";
+    std::string forward_name = "";
+    std::string backward_name = "";
 
-    std::string additional_data;
-    std::string color;
+    std::string additional_data = "";
+    std::string color = "";
     int sort = std::numeric_limits<int>::max();
 
-    CommercialMode* commercial_mode;
-    Network* network;
-    Company* company;
+    CommercialMode* commercial_mode = nullptr;
+    Network* network = nullptr;
+    Company* company = nullptr;
     nt::MultiLineString shape;
+    std::string opening_time = "", closing_time = "";
```

Comment:
Because this is easier, but yep, I should do it.

2015-02-10 11:49 GMT+01:00 Antoine D <notifications@github.com>:

> In source/ed/types.h
> <https://github.com/CanalTP/navitia/pull/879#discussion_r24403486>:
>
> >      nt::MultiLineString shape;
> > +    std::string opening_time = "", closing_time = "";
>
> why do you store that as string ?
> It is a time no ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CanalTP/navitia/pull/879/files#r24403486>.
>



-- 
Vincent Lara
06-82-01-63-46
lara.vincent@gmail.com